.gitignore

```diff
@@ -0,0 +1 @@
+coverage
```

Comment:
Oh, you gotta walk me through how to set this up. I'm not sure how git ignore works.